ray-operator/controllers/ray/common/service_test.go

```diff
@@ -425,6 +448,32 @@ func TestBuildServeServiceForRayService(t *testing.T) {
 	validateNameAndNamespaceForUserSpecifiedService(svc, serviceInstance.ObjectMeta.Namespace, expectedName, t)
 }
 
+func TestBuildServeServiceForRayCluster(t *testing.T) {
+	svc, err := BuildServeServiceForRayCluster(*instanceForServeSvc)
+	assert.Nil(t, err)
+
+	actualResult := svc.Spec.Selector[RayClusterLabelKey]
+	expectedResult := string(instanceForServeSvc.Name)
+	if !reflect.DeepEqual(expectedResult, actualResult) {
+		t.Fatalf("Expected `%v` but got `%v`", expectedResult, actualResult)
+	}
+
+	actualLabel := svc.Labels[RayServiceLabelKey]
```

Comment:
Maybe the K8s serve service for the RayCluster should not have this label.

Reply:
I do not see any harm in it